logpolicy/logpolicy.go

```diff
@@ -40,6 +41,23 @@ import (
 	"tailscale.com/version"
 )
 
+var getLogTargetOnce struct {
+	sync.Once
+	target string
```

Comment:
I commonly name this just `v`, since the "target" is stuttery redundant with "getLogTargetOnce". You can even remove "get" because it's the func that gets. This just holds. so `var logTargetOnce`. Then add a little comment like:

```
    v string // URL of logs server, or empty for default
```

Or attach to the variable if you prefer (above line 44)